src/main/js/app.js

```diff
@@ -2,12 +2,16 @@ const React = require('react');
 const ReactDOM = require('react-dom');
 
```

Comment:
The tutorial has all these: const React = require('react');
const ReactDOM = require('react-dom');
const client = require('./client'); 

Might be worth keeping in mind that we don't have the client here?

Reply:
Nice catch, i think it's because we have the client in the `PostBuilder` class instead as this is where it's used